pkgs/applications/editors/vscode/generic.nix

```diff
@@ -121,6 +121,9 @@ let
         --replace "/usr/bin/pkexec" "/run/wrappers/bin/pkexec" \
         --replace "/bin/bash" "${bash}/bin/bash"
       rm -rf "$packed"
+
+      # this fixes bundled ripgrep
+      chmod +x resources/app/node_modules/vscode-ripgrep/bin/rg
```

Comment:
Why are we not using the nixpkgs one?

Reply:
I'm not sure if the embedded one is not patched. Anyway it's already there.